app/src/main/java/com/example/newsportal/app/search/SearchNewsFragment.kt

```diff
@@ -44,7 +44,10 @@ class SearchNewsFragment : BaseFragment<FragmentSearchNewsBinding>(), FragmentWi
         }
 
```

Comment:
You have a bug in SearchNewsFragment. If we add news to bookmark, then bookmark icon will be visible. But after we go somewhere, return to SearchNewsFragment and find this news again, then it will be without bookmark icon